main.py

```diff
@@ -260,15 +264,30 @@ def usage(err = ""):
                     if '--tle' in [i[0] for i in optlist]:
                         tleFileName = [i[1] for i in optlist if i[0] == '--tle'][0]
 
-                    if not satName is None and not timeRec is None:
```

Comment:
Why was this deleted? Please keep it at least as a comment. If we decide to remove it, we will remove it properly.

Reply:
It is not deleted, it seems only to show as it is. I just have changed places with `not` part, because usually you check for None is: if satName is not None. So technically it was left as it is, I've changed only the visualization part.

Reply:
Github just duplicated the line below: `if satName is not None and not timeRec is None:`

Reply:
I didn't mean just that one line. I meant the visualization part too. It uses some part of the code elsewhere, and only this uses it. So, if we decide to get rid of this, that also should be deleted. So, just keep the old visualization code part in comment and we decide later. :)

Reply:
Okay.